refactor_debug.ipynb

```diff
@@ -0,0 +1,1657 @@
+{
```

Comment:
Rm this file?

Reply:
Yeah..sure. This was for the reviewer's testing...as I couldn't add tests yet. @kaavee315